podcasts/MultiSelectFooterView.swift

```diff
@@ -64,13 +64,16 @@ class MultiSelectFooterView: UIView, MultiSelectActionOrderDelegate {
     @IBOutlet var activityIndicator: ThemeLoadingIndicator!
     private var rightAction: MultiSelectAction?
     private var leftAction: MultiSelectAction?
+    private var numberOfEpisodes: Int
 
     override init(frame: CGRect) {
+        numberOfEpisodes = 0
         super.init(frame: frame)
         commonInit()
     }
 
     required init?(coder aDecoder: NSCoder) {
+        numberOfEpisodes = 0
         super.init(coder: aDecoder)
         commonInit()
     }
```

Comment:
Couldn't we set the default value on the var?

```suggestion
    private var numberOfEpisodes = 0

    override init(frame: CGRect) {
        super.init(frame: frame)
        commonInit()
    }

    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        commonInit()
    }
```